packages/driver/src/cy/commands/navigation.ts

```diff
@@ -50,32 +50,32 @@ const isValidVisitMethod = (method) => {
 const timedOutWaitingForPageLoad = (ms, log) => {
   debug('timedOutWaitingForPageLoad')
 
-  const anticipatedCrossOriginHref = cy.state('anticipatingMultiDomain')
+  const anticipatedCrossOriginHref = cy.state('anticipatingCrossOriginRequest')
 
   // Were we anticipating a cross domain page when we timed out?
```

Comment:
Do we want to update some of these comments as well?

Reply:
Yes, though i was leaving those for another PR. Let me see how many more changes navigation needs and if it affects other files.

Reply:
I checked on navigation, it'll require more changes that I'll make on another pr.